### ISSUE TITLE
Crash on device rotation with Google Map

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#15022]
### Describe your problem!

If I use unified map with Google maps and rotate the phone from portrait to landscape c:geo crashes. Offline maps seem to work.

### How to reproduce?

- Use Google:Map
- Rotate device to landscape

### Actual result after these steps?

Crash

### Expected result after these steps?

Normal operation

### Reproducible

Yes

### c:geo Version

2023.12.14-NB

**Comment by Lineflyer** [comment]
Log:
```
2023-12-15 00:13:22.256 25312-25312 Andro...time cgeo.geocaching  E  FATAL EXCEPTION: main
                                                                     Process: cgeo.geocaching, PID: 25312
                                                                     java.lang.IllegalStateException: Fragment GoogleMapsFragment{36476c8} (5c4744d4-67a2-4718-8ae3-1e1f25eaa1f7) did not return a View from onCreateView() or this was called before onCreateView().
                                                                     	at androidx.fragment.app.Fragment.requireView(Fragment.java:1964)
                                                                     	at cgeo.geocaching.unifiedmap.googlemaps.GoogleMapsFragment.setMapRotation(GoogleMapsFragment.java:279)
                                                                     	at cgeo.geocaching.unifiedmap.googlemaps.GoogleMapsFragment.onMapReady(GoogleMapsFragment.java:93)
                                                                     	at com.google.android.gms.maps.zzau.zzb(com.google.android.gms:play-services-maps@@18.2.0:1)
                                                                     	at com.google.android.gms.maps.internal.zzas.zza(com.google.android.gms:play-services-maps@@18.2.0:5)
                                                                     	at com.google.android.gms.internal.maps.zzb.onTransact(com.google.android.gms:play-services-maps@@18.2.0:3)
                                                                     	at android.os.Binder.transact(Binder.java:1200)
                                                                     	at m.bbe.c(:com.google.android.gms.policy_maps_core_dynamite@233610105@233610102065.573949206.573949206:8)
                                                                     	at com.google.maps.api.android.lib6.impl.cg.run(:com.google.android.gms.policy_maps_core_dynamite@233610105@233610102065.573949206.573949206:17)
                                                                     	at android.os.Handler.handleCallback(Handler.java:942)
                                                                     	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                     	at android.os.Looper.loopOnce(Looper.java:226)
                                                                     	at android.os.Looper.loop(Looper.java:313)
                                                                     	at android.app.ActivityThread.main(ActivityThread.java:8762)
                                                                     	at java.lang.reflect.Method.invoke(Native Method)
                                                                     	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
                                                                     	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
2023-12-15 00:13:22.256 25312-25312 cgeo         cgeo.geocaching  W  [main] UncaughtException
                                                                     java.lang.IllegalStateException: Fragment GoogleMapsFragment{36476c8} (5c4744d4-67a2-4718-8ae3-1e1f25eaa1f7) did not return a View from onCreateView() or this was called before onCreateView().
                                                                     	at androidx.fragment.app.Fragment.requireView(Fragment.java:1964)
                                                                     	at cgeo.geocaching.unifiedmap.googlemaps.GoogleMapsFragment.setMapRotation(GoogleMapsFragment.java:279)
                                                                     	at cgeo.geocaching.unifiedmap.googlemaps.GoogleMapsFragment.onMapReady(GoogleMapsFragment.java:93)
                                                                     	at com.google.android.gms.maps.zzau.zzb(com.google.android.gms:play-services-maps@@18.2.0:1)
                                                                     	at com.google.android.gms.maps.internal.zzas.zza(com.google.android.gms:play-services-maps@@18.2.0:5)
                                                                     	at com.google.android.gms.internal.maps.zzb.onTransact(com.google.android.gms:play-services-maps@@18.2.0:3)
                                                                     	at android.os.Binder.transact(Binder.java:1200)
                                                                     	at m.bbe.c(:com.google.android.gms.policy_maps_core_dynamite@233610105@233610102065.573949206.573949206:8)
                                                                     	at com.google.maps.api.android.lib6.impl.cg.run(:com.google.android.gms.policy_maps_core_dynamite@233610105@233610102065.573949206.573949206:17)
                                                                     	at android.os.Handler.handleCallback(Handler.java:942)
                                                                     	at android.os.Handler.dispatchMessage(Handler.java:99)
                                                                     	at android.os.Looper.loopOnce(Looper.java:226)
                                                                     	at android.os.Looper.loop(Looper.java:313)
                                                                     	at android.app.ActivityThread.main(ActivityThread.java:8762)
                                                                     	at java.lang.reflect.Method.invoke(Native Method)
                                                                     	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:604)
                                                                     	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1067)
```

**Comment by fm-sys** [comment]
Should already be fixed with today's nightly. Fix was part of PR  #15012

**Comment by Lineflyer** [comment]
Confirmed fixed in 2023.12.16-NB
